ibm/service/cis/resource_ibm_cis_mtls_app_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package cis_test
+
+import (
+	"fmt"
+	"testing"
+
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMCisMtlsApp_Basic(t *testing.T) {
+	name := "ibm_cis_mtls_app." + "test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheckCis(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCisMtlsAppBasic1("test", acc.CisDomainStatic),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(name, "url", "darunya.austest-10.cistest-load.com"),
```

Comment:
Add some update scenario's also as part of testig and import

Reply:
I'll ask @arpit-srivastava-ibm  to raise an issue. We'll add more test in xxx_testing.go later. We'll share the issue link. [Basic test case is added which will make sure successfully adding a resource and deleting it]

Reply:
Issue@ https://github.com/IBM-Cloud/terraform-provider-ibm/issues/3876